pkg/providers/tinkerbell/testdata/cluster_tinkerbell_multiple_node_groups.yaml

```
@@ -30,11 +35,21 @@ spec:
       machineGroupRef:
         name: test-md
         kind: TinkerbellMachineConfig
+      upgradeRolloutStrategy:
+        type: "RollingUpdate"
+        rollingUpdate:
+          maxSurge: 1
+          maxUnavailable: 0
       name: md-0
     - count: 1
       machineGroupRef:
         name: test-md
         kind: TinkerbellMachineConfig
+      upgradeRolloutStrategy:
+        type: "RollingUpdate"
+        rollingUpdate:
+          maxSurge: 1
+          maxUnavailable: 0
```

can you give the second worker node group a different `maxSurge` to make sure it's not using the same values for all
```suggestion
      upgradeRolloutStrategy:
        type: "RollingUpdate"
        rollingUpdate:
          maxSurge: 2
          maxUnavailable: 0
```